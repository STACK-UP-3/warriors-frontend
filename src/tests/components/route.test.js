import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AppRoute from '../../routes/index';

describe('Test <AppRoute/> componet', () => {
  const component = shallow(<AppRoute />);
  it('Should render AppRoute', () => {
    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
