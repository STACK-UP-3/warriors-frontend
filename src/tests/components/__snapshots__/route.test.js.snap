// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <AppRoute/> componet Should render AppRoute 1`] = `
<BrowserRouter>
  <div
    className="app"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/signup"
      />
    </Switch>
  </div>
</BrowserRouter>
`;
